### PR TITLE
Created ActivityStream page and its related classes

### PR DESCRIPTION
**Pull request by StepanRiazhkin**
Many things have been done!
Yet many things await to be done!

### CODE REVIEW

**Review by AaronQA — CHANGES_REQUESTED** [review]
I am sending a message about it
please make changes accordingly

File: src/test/java/com/crm/pages/LoginPage.java
Comment (StepanRiazhkin) [inline src/test/java/com/crm/pages/LoginPage.java:1]: Here I deleted unused imports (amount of 3)

File: src/test/java/com/crm/runners/CukesRunner.java
Comment (StepanRiazhkin) [inline src/test/java/com/crm/runners/CukesRunner.java:1]: Just deleted excessive spaces

File: src/test/java/com/crm/step_definitions/LoginPageStepDefs.java
Comment (StepanRiazhkin) [inline src/test/java/com/crm/step_definitions/LoginPageStepDefs.java:1]: Deleted my makeshift method for Logging in, now I use Mahrukh's method

File: src/test/resources/features/US_12_Appreciation.feature
Comment (StepanRiazhkin) [inline src/test/resources/features/US_12_Appreciation.feature:1]: Tested with all the data here and commented out some spares login data

File: src/test/java/com/crm/pages/ActivityStreamPage.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    public WebElement uploadElement;
+
+    @FindBy(xpath =
+            "//span[text()='Insert in text']"
+    ) // table where elements
+    public WebElement webTable_InsertTextButton_FirstElement;
+
+    @FindBy(xpath = "//tbody[@class='diskuf-placeholder-tbody']//tr[1]//td[5]") //
+    public WebElement webTable_DeleteButton_FirstElement;
+
```
Comment (StepanRiazhkin) [inline src/test/java/com/crm/pages/ActivityStreamPage.java:32]: The method I've created below "clicksAndCheckIfNoDisplayed" to process my two tests, where I should be adding uploaded files into the text, and where I should be deleting all the elements. In such way I had to decrease the implicit wait duration for a while, in order to do not wait for the element that has been already deleted. And after I've set the implicit duration back. And I had to "catch" the NoSuchElementException as there're no such elements after they have been removed in the while loop.

**Review by AaronQA — CHANGES_REQUESTED** [review]
can you change the login you used. I am sending a message about it

**Comment by StepanRiazhkin** [comment]
> can you change the login you used. I am sending a message about it

Fixed!

**Review by AaronQA — APPROVED** [review]

**Pull request merged**